### DISCUSSION TITLE
tsparticles with bootstrap 5

### DISCUSSION THREAD

**Discussion by rcorsari** [matteobruni/tsparticles#2672] [Q&A] [answered]
Hello almost everybody out there is using particles for the whole page background.

Instead, in a bootstrap project, I'd like particles as background for just the header section of the page

Mean I'll have particles right below the logo and the navigation menu

Can you kindly help with this need? 

Thank you

**Comment by matteobruni** [comment, marked as answer]
If you want to have particles inside a div instead of full screen you can read here: https://github.com/matteobruni/tsparticles/discussions/2591
